Resources/public/css/theme/modules/form.css

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+
+.pure-g [class *= "pure-u"],
```

Comment:
`.pure-g` and `.yui3-g` have nothing to do with forms, so you should left the original rules in app.css.